test/02_test/test/spam_test.py

```diff
@@ -1,9 +1,39 @@
+from __future__ import print_function
+import os
+import sys
 from unittest import TestCase
 
 import spam
 
 
+def path_contains(parent, child): 
```

Comment:
```suggestion
def path_contains(parent, child): 
    ''' returns True if `child` is inside `parent`.

    Works around path-comparison bugs caused by short-paths on Windows e.g.
    vssadm~1 instead of vssadministrator
    '''
```